deepchecks/utils/distribution/preprocessing.py

```diff
@@ -215,3 +215,14 @@ def value_frequency(x: Union[List, np.ndarray, pd.Series]) -> List[float]:
     total_occurrences = len(x)
     values_probabilities = list(map(lambda n: n / total_occurrences, x_values_counter.values()))
     return values_probabilities
+
+
+def one_hot_batch(batch: Union[np.ndarray, List], num_classes: int) -> np.ndarray:
```

Comment:
see the function on _transform_to_multi_label_format on scorers.py 
I think they do the same thing so you can remove one of them (I prefer this location better)

Reply:
_transform_to_multi_label_format doesn't take number of classes so it isn't compatible with the case of batches - a class might not appear in the batch but exist in other batches

Reply:
it takes a classes argument which is an extension of num_classes to a list of the class names